Readme.md

```diff
@@ -0,0 +1,73 @@
+# SellEverywhere
+
+## Overview
+We want to create a hardware shop using popular technologies in a friendly team
+(MKdir). Examples are any store selling goods. Current idea is to implement an
+admin panel and user panel. The basic interface of the store was designed in [Figma](https://www.figma.com/file/Re4tXW7aNCWOzed7wI1GKe/dirStore).
+
+### Basic flow
+![Basic flow](./docs/basic_flow.png)
+
+### The application must have the following functionality from the administrator perspective:
```

Comment:
think about Variants of products.
Like one product in different color scheme, or in different size

Reply:
This project is about selling hardware, so we will think about technical characteristics

Reply:
technical characteristics will cause variants or not?

Reply:
Characteristics will be different for a specific category